app/assets/stylesheets/application.scss

```diff
@@ -17,21 +17,28 @@
 @import "bootstrap";
 
 body {
-  background-color: #e5e8e0;
+  background-color: #e0e0e0;
 }
 
-.container-fluid { max-width: 80%;
-
+.container-fluid {
+  max-width: 80%;
 }
 
-img {width: 100%;
-
+img {
+  width: 100%;
 }
 
 .featured {
   background-color: #c3a17d;
-  h3{color: white;
+  h3 {color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.